app/services/invoices/customer_usage_service.rb

```diff
@@ -60,8 +62,10 @@ def customer(customer_id: nil)
       )
     end
 
-    def subscription
-      @subscription ||= customer.active_subscription
+    def find_subscription(subscription_id)
+      return nil unless customer
```

Comment:
Since you are now using the safe travel operator, you can also remove this guard

Reply:
fixed